sway/desktop/render.c

```diff
@@ -1018,7 +1018,7 @@ void output_render(struct render_context *ctx) {
 
 	if (!pixman_region32_not_empty(damage)) {
 		// Output isn't damaged but needs buffer swap
-		goto renderer_end;
```

Comment:
Hm, I don't think we even need to render software cursors in that case? So maybe we can just `return`?

Reply:
Software cursors submit damage on their own... Oh, but that should just go into the damage ring. I think a return here should be fine.